packages/salesforcedx-vscode-lwc/src/testSupport/commands/forceLwcTestDebugAction.ts

```diff
@@ -42,6 +42,7 @@ export function getDebugConfiguration(
     cwd,
     runtimeExecutable: command,
     args,
+    resolveSourceMapLocations: ['**', '!**/node_modules/**'],
```

Comment:
The new debugger needs this arg so it doesn't show a warning message about missing source map for rollup (warning only, doesn't affect debugging). This doesn't affect the old debugger.